.github/workflows/test-k3s.yml

```diff
@@ -34,4 +34,4 @@ jobs:
         # NOTE: "PATH=$PATH" preserves the default user $PATH. This is needed to maintain the version of go installed
         #       in a previous step. This test run will use Zarf to create a K3s cluster, and a brand new cluster will be
         #       used for each test
-        run: sudo env "PATH=$PATH" TESTDISTRO=k3s make test-e2e
+        run: sudo env "PATH=$PATH" CI=true APPLIANCE_MODE=true make test-e2e
```

Comment:
I'm surprised we needed this CI flag but my eyes don't lie from the last tests 🤷 

Although, is there are reason we're only providing it for k3s?

Reply:
Yeah sudo was dropping the env variable.  Not totally sure why.  I thought sudo env fixed that but I might be misremembering.